clang/lib/Driver/Driver.cpp

```diff
@@ -4742,6 +4774,28 @@ class OffloadingActionBuilder final {
     return false;
   }
 
+  void addDeviceDependencies(ActionList &LinkerInputs) {
```

Comment:
Please add some more comments - quite tricky place. The LinkerInputs are host inputs, and LA below is a host binary, correct?

Reply:
Done.

Reply:
thanks!